scripts/background.js

```diff
@@ -3,20 +3,22 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   if (
     changeInfo.status === "complete" &&
     tabUrl &&
-    tabUrl.includes("linkedin.com/jobs/search")
+    tabUrl.includes("linkedin.com/jobs/view")
   ) {
     chrome.scripting.insertCSS({
       target: { tabId: tabId },
-      files: ["css/search.css"],
+      files: ["css/view.css"],
     });
   } else if (
-    changeInfo.status === "complete" &&
-    tabUrl &&
-    tabUrl.includes("linkedin.com/jobs/view")
+    (changeInfo.status === "complete" &&
+      tabUrl &&
+      tabUrl.includes("linkedin.com/jobs/search")) ||
+    tabUrl.includes("linkedin.com/jobs/collections") ||
+    tabUrl.includes("linkedin.com/jobs")
```

Comment:
Needed to include this route since it's the main jobs page. Also added collections since that was no longer being accounted for.